static/app/views/issueDetails/streamline/eventGraph.tsx

```diff
@@ -288,6 +298,11 @@ export function EventGraph({
     onReleaseClick: handleReleaseLineClick,
   });
 
+  const flagSeries = useFlagSeries({
+    event,
+    flags,
+  });
```

Comment:
This hook is now split up into 2: `useIntersectionFlags` which does the data fetching + intersection logic and `useFlagSeries` which creates the chart series.

We need the data fetching separated for the release bubbles.